kv/kv.go

```diff
@@ -217,6 +220,8 @@ type ResultSubset interface {
 	GetStartKey() Key
 	// GetExecDetails gets the detail information.
 	GetExecDetails() *execdetails.ExecDetails
+	// MemSize returns how many bytes of memory this result use for tracing memory usage.
+	MemSize() int
```

Comment:
`int` is a little small to store memory size, it can only represent 2^32 bytes, which is 4GB. How about using `int64`?